dist/message.js

```diff
@@ -0,0 +1,70 @@
+var RiseVision = RiseVision || {};
+RiseVision.Common = RiseVision.Common || {};
+
+RiseVision.Common.Message = function (mainContainer, messageContainer) {
+  "use strict";
+
+  var _active = false;
+
+  function _init() {
+    try {
+      messageContainer.style.height = mainContainer.style.height;
```

Comment:
@stulees curious, isn't this height always going to be the height of the iframe the widget is in? Basically, I can't justify having the `mainContainer` as a variable for simply extracting the height. Thanks.